tests/conftest.py

```diff
@@ -441,3 +444,58 @@ def __init__(self, *args, **kwargs):
             super().__init__(*args, **kwargs)
 
     monkeypatch.setattr("ert.cli.main.EvaluatorServerConfig", MockESConfig)
+
+
+QSTAT_HEADER = (
+    "Job id                         Name            User             Time Use S Queue\n"
+    "-----------------------------  --------------- ---------------  -------- - ---------------\n"
+)
+QSTAT_HEADER_FORMAT = "%-30s %-15s %-15s %-8s %-1s %-5s"
+
+
+@pytest.fixture
+def create_mock_flaky_qstat(monkeypatch, tmp_path):
+    bin_path = tmp_path / "bin"
+    bin_path.mkdir()
+    monkeypatch.chdir(bin_path)
+    monkeypatch.setenv("PATH", f"{bin_path}:{os.environ['PATH']}")
+    yield _mock_flaky_qstat
```

Comment:
(does it make sense to `yield` here when nothing is done after the `yield`?) 

Should this and the next function just be merged?

Reply:
hmm, you are using this fixture as a plain function, you might not need the `@pytest.fixture` decorator.

Reply:
I am using the monkeypatch fixture to temporarily change the PATH variable, and I do not think I can do it this way without having it as a function-yielding fixture.

Reply:
ah! right